security/entity_provider.rst

```diff
@@ -169,6 +169,12 @@ forces the class to have the five following methods:
 
 To learn more about each of these, see :class:`Symfony\\Component\\Security\\Core\\User\\UserInterface`.
 
+.. caution::
+
+    Do not actually implement ``eraseCredentials`` when you load your users directly
```

Comment:
Yes that's the obvious thing, I don't want to clear my database stuff. The thing is that I wasn't aware that `eraseCredentials` was executed every request, so somehow exactly that should be mentioned. This could also be deprecated on the interface, and transformed into an event?